control/registration/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "ignore.go",
+        "registration.go",
+        "summary.go",
+    ],
+    importpath = "github.com/scionproto/scion/control/registration",
```

Comment:
registration is a bit vague and can mean many things.

Maybe it makes sense to move this to the segreg package (directory already exists)

(no strong opinion)